### DISCUSSION TITLE
ARM32 support?

### DISCUSSION THREAD

**Discussion by kristibektashi** [namazso/SecureUxTheme#179] [Ideas]
Maybe you could add support for ARM32 versions of Windows, such as Windows 10 build 15035 and jailbroken versions of Windows RT

**Comment by namazso** [comment]
This should've been an issue, but no, I'd rather not. Toolchains aren't very polished for that target, since it's dead.

**Reply by kristibektashi** [reply]
Would you accept a PR if someone were to implement the support for you? (Assuming it doesn't break anything else)

**Reply by namazso** [reply]
No, because I don't wish to maintain it with a different toolchain, and the current one no longer supports ARM32. Just make a fork, it's not like updates will ever be needed, that platform is as dead as IA64.
